examples/revert/Revert.hs

```diff
@@ -1,18 +1,20 @@
 module Main where
 
 import Data.Text                  (Text)
+import Control.Monad.Reader       (runReader, runReaderT)
 import Control.Monad.State.Strict (evalState)
 import Hedgehog hiding            (checkParallel)
 import Hedgehog.Internal.Property (GroupName(..), PropertyName(..))
 
 import EVM (VM, VMResult(..), Error( Revert ))
 
+import Echidna.Config (defaultConfig)
 import Echidna.Exec
 import Echidna.Solidity
 
 main :: IO ()
-main = do (v,a,ts) <- loadSolidity "solidity/revert.sol" Nothing Nothing
-          let prop t = (PropertyName $ show t, ePropertySeq (`checkRTest` t) a v 10)
+main = do (v,a,ts) <- runReaderT (loadSolidity "solidity/revert.sol" Nothing) defaultConfig
```

Comment:
Can we add some `withDefaultConf :: MonadReader Config m => m a -> a` to account for this pattern of usage, then just put it left of `do` here?